examples/asr/transcribe_speech.py

```diff
@@ -149,12 +156,24 @@ def main(cfg: TranscriptionConfig) -> TranscriptionConfig:
     asr_model = asr_model.eval()
     partial_audio = False
 
+    # we will adjust this flag is the model does not support it
+    compute_langs = cfg.compute_langs
+
     # Setup decoding strategy
     if hasattr(asr_model, 'change_decoding_strategy'):
         # Check if ctc or rnnt model
         if hasattr(asr_model, 'joint'):  # RNNT model
-            asr_model.change_decoding_strategy(cfg.rnnt_decoding)
+            if isinstance(asr_model.decoding.tokenizer, AggregateTokenizer):
```

Comment:
I was just considering, would it be bad if we made return of the hypothesis the default thing and ignore the return of pure text case ? I want to keep this script as simple as possible, and it's getting messy with all these ifs and else's. 

The user can set the flag and run inference, let the code hard crash if it's not supported in the lower level call. Skip this if else and logging warning + explicitly updating the config etc.

Reply:
done